web/components/InnovFooter.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { ReactElement } from "react";
+import { Footer } from "@/display-content/Footer";
+import { Icon } from "@/components/njwds/Icon";
+
+
+export const InnovFooter = (): ReactElement => {
+    return (
+        <div className="usa-identifier">
+        <div className="display-flex flex-column margin-x-auto fac padding-top-2 footer-custom-desktop">
+
+        <section className="usa-identifier__section usa-identifier__section--usagov " aria-label="U.S. government information and services">
+            <div className="usa-identifier__container">
+                
+        <p className="footer-container footer-container-mobile">
```

Comment:
instead of having both these classes, you should be able to just use a single class, and use css media queries to describe the mobile behavior. let me know if you have questions on this